src/Build/BackEnd/Components/Scheduler/Scheduler.cs

```diff
@@ -144,7 +143,10 @@ internal class Scheduler : IScheduler
         /// <summary>
         /// Flag used for debugging by forcing all scheduling to go out-of-proc.
         /// </summary>
-        internal bool ForceAffinityOutOfProc { get; private set; }
+        internal bool ForceAffinityOutOfProc
```

Comment:
It'd be nice if you could cache this, as it was before, but invalidate it when we start a new build. Not an important difference, though.